recipes/graphviz/2.44.1/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans import ConanFile, tools, CMake
+import os
+
+class Graphviz(ConanFile):
+    name = "graphviz"
+    description = "Conan installer for Graphviz"
+    topics = ("conan", "graphviz", "dot", "doxygen", "installer")
+    url = "https://gitlab.com/graphviz/graphviz/"
+    homepage = "https://graphviz.org/"
+    license = "Eclipse Public License - v 1.0"
+    generators = "virtualrunenv", "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+
+    _cmake = None
+    _source_subfolder = "source_subfolder"
+    short_paths = True
+
+    def build_requirements(self):
+        if tools.os_info.is_linux and self.settings.os == "Linux":
+            self.build_requires("bison/3.5.3")
+            self.build_requires("expat/2.2.9")
+            self.build_requires("flex/2.6.4")
+            self.build_requires("fontconfig/2.13.91")
+            self.build_requires("freetype/2.10.2")
+            self.build_requires("libpng/1.6.37")
+            self.build_requires("pixman/0.40.0")
+            self.build_requires("zlib/1.2.11")
+
+        if tools.os_info.is_windows and self.settings.os == "Windows":
+            if not tools.get_env("CONAN_BASH_PATH"):
+                self.build_requires("msys2/20200517")
+            self.build_requires("strawberryperl/5.30.0.1")
+            self.build_requires("winflexbison/2.5.22")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        folder_name = "%s-%s" % (self.name, self.version)
+        os.rename(folder_name, self._source_subfolder)
+        # Get windows dependencies from official repo
+        if tools.os_info.is_windows and self.settings.os == "Windows":
+            git = tools.Git(folder=os.path.join(self._source_subfolder, "windows", "dependencies", "libraries"))
+            git.clone("https://github.com/ErwinJanssen/graphviz-windows-dependencies.git", branch="141d3a21be904fa8dc2ae3ed01d36684db07a35d", shallow=True)
```

Comment:
Please use cci for all dependencies.

Reply:
I was able to get all dependencies except for rxspencer. I had a quick try on building it myself, but unless I use the provided lib I get compile errors. I need to have a deeper look at this.